data/free worlds intro.txt

```diff
@@ -2020,11 +2020,21 @@ mission "Defend Sabik"
 	on complete
 		dialog `The Navy has been driven off, but at a great cost. You should probably visit the spaceport and see if Tomek is there, or someone else who can tell you what to do next.`
 
-mission "FWCarrier License"
+mission "FW License"
```

Comment:
This mission can actually be deleted with `set "license: Militia"` going under the `on offer` of "Defend Sabik", as the only reason the "FWCarrier License" existed was for forward compatibility of old saves. Now though, the compatibility of old saves is covered by "FW License Fix", so there's no need for the Militia license to be offered by a separate mission.